### PR TITLE
Travis CI enabled

### PR DESCRIPTION
**Pull request by johnsyweb**
#### Context

Since we released to RubyGems, a few little niggles have appeared.

#### Change

 - Bump Ruby, RubyGems & Bundler in Travis CI config
 - Update build badge to Travis CI
 - Update change log

#### Considerations

Will release as 0.3.1

#### Cc

@orien, @joesustaric

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.2
-before_install: gem install bundler -v 1.16.1
+  - 2.4.5
```
Comment (joesustaric) [inline .travis.yml]: no 2.3 or 2.2 ? I think that if this project is trying to promote better gem lib freshness.. I would suspect those projects might be not so fresh with their ruby versions as well.. just a thought..

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+  - 2.5.3
+before_install:
+  - gem update --system
+  - gem install bundler -v 1.17.1
```
Comment (joesustaric) [inline .travis.yml]: Why do we need to specify the version of bundler?

Comment (johnsyweb) [reply .travis.yml]: We don't. Addressed in 0e090f3.

File: .travis.yml (hunk 1 above)
Comment (johnsyweb) [reply .travis.yml]: Attempted in a9487b1. Fixed in 6c27f57.

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ to get regular dependency updates into production.
   - Vulnerability advisory information using [bundler-audit](https://github.com/rubysec/bundler-audit)
   - Links to the home page, source code and change log (where available) of each gem
 
-## Development status [![Build status](https://badge.buildkite.com/d7db34f910131ff2a03d31dcc0ee960a3bc5f0df2c42ec4eb4.svg?branch=master&style=flat-square)](https://buildkite.com/envato-marketplaces/unwrappr)
+## Development status [![Build Status](https://travis-ci.org/envato/unwrappr.svg?branch=master)](https://travis-ci.org/envato/unwrappr)
```
Comment (orien) [inline README.md]: I'd be nice if they had a flat style like the other badges in the readme.

**Pull request merged**